blivet/devicelibs/dasd.py

```diff
@@ -114,6 +114,10 @@ class dasd_info(ctypes.Structure):
 
 def is_ldl_dasd(device):
     """Determine whether or not a DASD is LDL formatted."""
+    statusfile = "/sys/block/%s/device/status" % (device,)
+    if not os.path.isfile(statusfile):
+        return False
```

Comment:
Only DASD have `/sys/class/block/<sysname>/device/status` ? It seems like either the commit message is inaccurate or the test is too inclusive.